configs/custom/classification/lenet/epoch_trainer.yaml

```diff
@@ -0,0 +1,13 @@
+_BASE_:
+  [
+    "../../../datasets/cifar10.yaml",
+    "../../../networks/classification/lenet.yaml",
+    "../../../pytorch_trainer/epoch_trainer.yaml",
+  ]
```

Comment:
Inheritance Hierarchy is correct.
But the naming of leaf file should be the network info.
For example, `fcos_r50_caffe_fpn_gn-head_mstrain_640-800_2x_coco.yaml`.
But it doesn't have to be fixed.
Just make a note of naming logic.